atom/browser/ui/certificate_trust_win.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2013 GitHub, Inc.
```

Comment:
Hmm, I'm not seeing a public API in https://cs.chromium.org/chromium/src/net/ssl/client_cert_store.h?dr=C or https://cs.chromium.org/chromium/src/net/ssl/client_cert_store_win.h?dr=C to add a certificate to the store.